### DISCUSSION TITLE
Shaking table for simulating earthquake with unsaturated soil

### DISCUSSION THREAD

**Discussion by Mangwuha** [Anura3D/Anura3D_OpenSource#255] [Q&A]
Dear all,
I am currently attempting to simulate a shaking table test. Previously, I successfully implemented a slope soil material under dry seismic conditions. However, when I replaced the soil with unsaturated soil that has suction, the model failed to compute correctly and reported an error indicating particles were exiting the grid. Given that the soil was substituted with unsaturated material, I also established liquid fixity and seepage face area parameters. All other parameter settings and boundary conditions remain consistent with those of my previously successful model. Does anyone have insights into what might have gone wrong?
Thank you for helping me.
![error](https://github.com/user-attachments/assets/51f6fcef-0401-44ec-9ed2-cc19a1e16ad7)
![material](https://github.com/user-attachments/assets/896f3a9d-514e-4e6a-962a-48bd3064cacd)
![parameters](https://github.com/user-attachments/assets/06cebd2d-c6c7-4e72-9a97-f60f3dc72c38)
![solid fixity](https://github.com/user-attachments/assets/8731128c-ea64-41e3-9b88-f023fde05139)
![liquid fixity](https://github.com/user-attachments/assets/b16f693d-cb9b-4117-afe0-ff475afb659c)
![PV](https://github.com/user-attachments/assets/9c406175-ea9f-43e2-ae6e-78bb4172945f)
![seepage  face area](https://github.com/user-attachments/assets/eed26b54-2275-48a0-bb15-933697e5897d)
![contact](https://github.com/user-attachments/assets/98f5c4ec-0259-4f58-8ae4-c20e53fbf884)
![5a76ff098ebce37454efa63cae1a5c6](https://github.com/user-attachments/assets/caf97238-fd89-4f72-bc95-cede84edd3f0)
![26dce2324b91f261fff6606708d6168](https://github.com/user-attachments/assets/21c642e3-dbb4-4be4-838f-ed458181f98b)
![43977b749443d2dcde2af2fbadfe2a1](https://github.com/user-attachments/assets/123cbd7c-c05d-4b47-9dc0-c9516c2bc5c6)

**Comment by 6cs4u** [comment]
Hello, may I ask how your seismic wave boundary is applied? I couldn't find any examples of simulating earthquakes in the manual and would like to learn more

**Reply by Mangwuha** [reply]
Hello，
Seismic waves are input as PRESCRIBED VELOCITY, and the time history points of seismic waves can be written in. PVF files are used in CPS files. The PVF file replaces a certain fixed value you set in the GID.
